yogstation/code/modules/mob/living/simple_animal/drone/interaction.dm

```diff
@@ -0,0 +1,71 @@
+/////////////////////
+//DRONE INTERACTION//
+/////////////////////
+//How drones interact with the world
+//How the world interacts with drones
+//Sounds like the pitch for a decent movie tbh
+
+/mob/living/simple_animal/drone/proc/yogs_drone_hack(hack, clockwork) // Basically an edited version of update_drone_hack.
+//Based on what update_drone_hack was on 19th of June, 2018.
+	if(!istype(src) || !mind)
```

Comment:
Why the ``!istype(src)`` check?

Reply:
I don't know, actually. It was in update_drone_hack already when I pulled it out.